common/config/rush/pnpm-config.json

```diff
@@ -6,7 +6,8 @@
     "ignoreMissing": ["@babel/core", "@types/node", "@typescript-eslint/parser"]
   },
   "globalOverrides": {
-    "mysql2": "^3.9.8" // https://github.com/advisories/GHSA-4rch-2fh8-94vw>azurite>mysql2, https://github.com/advisories/GHSA-pmh2-wpjm-fj45
+    "mysql2": "^3.9.8", // https://github.com/advisories/GHSA-4rch-2fh8-94vw>azurite>mysql2, https://github.com/advisories/GHSA-pmh2-wpjm-fj45
+    "braces": "^3.0.3"  // https://github.com/advisories/GHSA-grv7-fg5c-xmjg
```

Comment:
Can you add how we are getting this dependency? Like look at the above

Reply:
Added